packages/react/src/themes/teams/components/Icon/svg/icons/index.ts

```diff
@@ -13,6 +13,7 @@ import calendar from './calendar'
 import call from './call'
 import callControlPresentNew from './callControlPresentNew'
 import callControlStopPresentingNew from './callControlStopPresentingNew'
+import callDialPad from './callDialpad'
```

Comment:
case rules should be consistent between import and file name - given that `dialpad` in file name is one word (`pad` part is not capitalised), the same should stay for the imported variable. This significantly simplifies automated refactoring scenarios:

```jsx
import callDialpad from './callDialpad'
```